cockroachdb-in-comparison.md

```diff
@@ -2,3 +2,20 @@
 title: CockroachDB in Comparison
 toc: false
 ---
+
+| | CockroachDB | MySQL | Postgre | Cassandra | HBase | MongoDB | Riak 
+--|--|--|--|--|--|--|--
+Automated Scaling & Repair| Yes | No | No |  |  | No | Yes 
+Synchronous Replication | Yes | No | No |  |  | No |  
+Automatic Failover | Yes | No | No |  |  |  | Yes 
+Read from Any Replica | No |  |  |  |  | Yes |  
```

Comment:
CockroachDB does have "Read from any Replica" but it's eventually consistent.

Here's a link to my estimation of the comparison matrix: https://docs.google.com/a/cockroachlabs.com/spreadsheets/d/1ZHhLkku2CJkUhyoAT_NUqOQmAPDC2cJ_2mfK4NXcarw/edit?usp=sharing